### PR TITLE
hotFix/colors

### PR DESCRIPTION
**Pull request by luisarmandodv**

### CODE REVIEW

File: MH-FiorellaTheme5/config.ru [hunk 1]
```diff
@@ -0,0 +1,30 @@
+require "rack"
```
Comment (JuanCrg90) [inline MH-FiorellaTheme5/config.ru]: Why you have this file in this Pull Request?

File: MH-FiorellaTheme5/source/categories.html.erb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<div class="container">
```
Comment (JuanCrg90) [inline MH-FiorellaTheme5/source/categories.html.erb]: Why you have this file in this Pull Request?

File: MH-FiorellaTheme5/source/categories.html.erb [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+  <%= partial "partials/_top-navigation" %>
+
```
Comment (JuanCrg90) [inline MH-FiorellaTheme5/source/categories.html.erb]: Extra blank space?

File: MH-FiorellaTheme5/config.ru (hunk 1 above)
Comment (luisarmandodv) [reply MH-FiorellaTheme5/config.ru]: This files are necessary to deploy to heroku, and they were in my local repository and i decided to added them

File: MH-FiorellaTheme5/source/categories.html.erb (hunk 2 above)
Comment (luisarmandodv) [reply MH-FiorellaTheme5/source/categories.html.erb]: Oh... I did a checkout in master (MH-theme5), and when a did a pull that changes were brought.

File: MH-FiorellaTheme5/source/categories.html.erb (hunk 3 above)
Comment (luisarmandodv) [reply MH-FiorellaTheme5/source/categories.html.erb]: yep... I will remove it

File: MH-FiorellaTheme5/source/stylesheets/pdp-styles.css [hunk 4]
```diff
@@ -34,18 +34,19 @@
   width: 50px;
 }
 
-.product .colors-bouquet .pink-circle {
-  background-color: pink;
+.product .colors-bouquet span:nth-child(1) {
```
Comment (JuanCrg90) [inline MH-FiorellaTheme5/source/stylesheets/pdp-styles.css]: mejor utiliza el approach que tenias aqui, esto puede dar problemas de mantenimiento a futuro

**Pull request merged**